Orange/classification/_tree_scorers.pyx

```diff
@@ -0,0 +1,403 @@
+#cython: embedsignature=True
+#cython: boundscheck=False
+#cython: wraparound=False
+#cython: infer_types=True
+#cython: cdivision=True
+
+
+import numpy as np
+numpy = np
```

Comment:
Why `import numpy as np` to then call `numpy.empty()`? Use `np.empty()` or just `import numpy`?